llama_stack/providers/adapters/safety/bedrock/config.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Optional
+
+from llama_models.schema_utils import json_schema_type
+from pydantic import BaseModel, Field
+import boto3
+
+
+@json_schema_type
+class BedrockShieldConfig(BaseModel):
+    """Configuration information for a guardrail that you want to use in the request."""
+
+    boto_client: Optional[boto3] = Field(
```

Comment:
the config is a data object, no code object should be here. you should instantiate the boto client in the initialize method of your adapter -- or if the client needs API keys then only at the time of the request when you can read these keys.